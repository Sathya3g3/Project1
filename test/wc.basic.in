This is a basic test case.
It checks if word_count_analysis counts lines, words, and characters correctly.
