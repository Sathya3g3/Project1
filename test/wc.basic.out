2 16 107 test/wc.basic.in